app/styles/app.scss

```diff
@@ -14,6 +14,13 @@ body:not(.simplified_wrapper):not(.container_stripped) {
 #ember-testing {
   transform: translateZ(0);
 }
+
+@import "election-countdown/election-countdown-promo";
+.election-promo {
```

Comment:
Maybe a little nit-picky but It feels weird to me for this code to be directly in the app.scss even though it's small.  Everything else looks good.

Reply:
100% agreed, I'm going to clean this up in a separate PR because we're under a time crunch